### DISCUSSION TITLE
Unable to reach server when transmission is running on it?

### DISCUSSION THREAD

**Discussion by colethegamer123** [transmission/transmission#2518] [Q&A]
Hello Dear Transmission,

I have a problem since months, and im struggling to solve it anyhow. 
I already asked plenty of discord channels about Linux & Transmission and on Transmission forum also.

**I have a transmission running inside Docker right now. Im using this to download Torrents.**

Anytime I download a torrent after a few seconds I cannot reach anymore my whole Computer. Everything running on this Server dies and cannot be reached. Even if it can be reached for example with SSH it takes 20 minutes to login.

A lot of things running on this server for example this handles my whole Smart Home, Media Server, Password Server etc...

What I tried:
 - Reinstall Transmission 3.0
 - Deupgrade Tranmission to 2.94
 - Deupgrade Transmission to 2.9
 - Install Transmission 3.0 inside Docker Container
 - Install Transmission 2.94 inside Docker Container
 - Reinstall the whole Operating System.
 - Upgrade Operating System from Debian9 to Debian10
 - Change all the settings inside Transmission
 - Change Port in Transmission
 - Set Download and Upload speed to 1kbit/s maximum
 - Change max Peer connections to 1
 - Buy a new Router
 - Change UTP Cable
 - Reconfigure my whole Network inside Router and Reset it

If you have any other Idea what should I try please tell me. I cannot find any logical answer.
I'm downloading to an external HDD.

One more thing I didnt tried, buy a new network card, what is a really cheap 1$ Aliexpress network card.

Thank you very much,

**Comment by colethegamer123** [comment]
Only thing I see is my CPU is going up to 80% after I start anything and the ntfs.mount process going up to 60% after Transmission is going down.

**Comment by 1265578519** [comment]
ulimit -n 1048576
echo "* soft nofile 1048576" >> /etc/security/limits.conf
echo "* hard nofile 1048576" >> /etc/security/limits.conf
reboot
试一下，设置完成后此时可以登录ssh

分析，默认值ulimit为1024，Tranmission 占用了所有的ulimit通道，导致系统爆掉，可以增大系统的限制来解决该问题

**Comment by colethegamer123** [comment]
This should be go into docker or into host? Thanks

**Comment by 1265578519** [comment]
> 这应该进入docker还是进入主机？谢谢

docker和主机节点，同时需要进行设置

**Comment by colethegamer123** [comment]
Still same problem.

**Comment by 1265578519** [comment]
> 还是同样的问题。

那我就不知道了，，我也遇到相同的问题，修改ulimit后就解决了

**Comment by JEZHDK** [comment]
Hi, i had similar issues.

Check /var/logs/kern.log for clues on the network card/interface halting.

A very common intel chip has a bug that seems to be difficult to fix, buth under heavy it can stop working.
I noticed my hdd led was blinking, as if there was activity on the system. Try check this.

**Comment by pataquets** [comment]
Try using `host` networking mode for the container. Warning: no port mappings possible, it will share your host net iface instead of creating its own, isolated interface. See docs for details.
Also, a possible workaround to ease stress on all possible TCP breaking points would be feature request #1519.

**Comment by axvpast** [comment]
Usually when transmission finish torrent it copy (not rename) your torrent files from Incomplete directory to Download.
Main question about your docker-transmission configuration. Where is physically located Incomplete directory? If it is located inside docker container that means all your ( big?) files is stored in the docker's home, in /var/lib/docker/...
So when transmission finish data transfer it copy data from some overlay fs from /var/lib/docker/.. to Downloads.
Freezing behavior usually happens when you got OOM (cannot login... because ssh cannot allocate memory for bash, but you can ping host) or when OS try to flush caches to HDD but HDD stuck. It can stuck because it is slow or because it is out of power (disk start current can be 2A of may be more!). In that case HDD can be few times rebooted (internally).
ToDo, ToCheck:
1. Check Free RAM
2. check ping in dead situation
3. check transmission docker construction - where is physically located Incomplete and Download dirs
4. disable 'start when added' checkbox in transmission, add torrent and change it's files location to Download folder (it can avoid copy files from Incomplete to Download directory) and start torrent.
5. try to copy big files from your external drive to your external drive
6. check your HDD specification regarding required power consumption, especially in case of speenup. Do you have enough power to supply it?

**Comment by foxli91** [comment]
> ulimit -n 1048576 echo "* soft nofile 1048576" >> /etc/security/limits.conf echo "* hard nofile 1048576" >> /etc/security/limits.conf reboot 试一下，设置完成后此时可以登录ssh
> 
> 分析，默认值ulimit为1024，Tranmission 占用了所有的ulimit通道，导致系统爆掉，可以增大系统的限制来解决该问题

大佬请教一下，使用Linux centos挂机，开启transmission下载一段时间之后内存的cache疯狂占满这个怎么解决呢，我目前的办法是设置脚本隔一段时间检测一下内存然后达到一定的数值使用命令清空的，请问您是怎么解决的呢？

**Comment by ckerr** [comment]
This appears to be a configuration or home networking problem rather than a bug in Transmission itself, as evidenced by the fact that there are a lot of people running transmission-daemon remotely without experiencing this problem.

I don't think the original question has been answered, but I also don't think it's a Transmission bug to be fixed, so I'm converting this from a bug report to a discussion question.
